contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -233,8 +233,35 @@ TsqlFunctionConvert(TypeName *typename, Node *arg, Node *style, bool try, int lo
 Node *
 TsqlFunctionIdentityInto(TypeName *typename, Node *seed, Node *increment, int location)
```

Comment:
will identity function be called in this case ?

Reply:
The actual functions wont be invoked, but we need the typename, seed and increment value. I can make a single function

Reply:
Removed two extra function int and smallint and just kep one, bigint which will take care of all numeric cases